ruby/src/main/java/com/google/protobuf/jruby/RubyMessage.java

```diff
@@ -246,16 +246,15 @@ public IRubyObject methodMissing(ThreadContext context, IRubyObject[] args) {
     public IRubyObject dup(ThreadContext context) {
```

Comment:
Thanks for finding these issues with the JRuby implementation! It was a contribution and I had thought I reviewed it carefully but it seems a few things slipped through :-)